.github/workflows/CI.yml

```diff
@@ -0,0 +1,151 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+  schedule:
+    # Nightly tests run on master by default:
+    #   Scheduled workflows run on the latest commit on the default or base branch.
+    #   (from https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule)
+    - cron: "0 0 * * *"
+
+
+jobs:
+  unix:
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.name }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Linux CPU CUDA 10.0 Python 3.6
+            python-version: "3.6"
+            os: ubuntu-latest
+            gcc-version: "9"
+            cuda-version: "10.0"
```

Comment:
Is any reason why not to test with the latest CUDA?

Reply:
Not really. Installation size, if anything. We can change that or add multiple matrix entries for different CUDA versions.

Reply:
I think, only the latest CUDA version is enough. Also `conda install -c conda-forge openmm-plumed` installs CUDA 11.

Reply:
The CUDA versions depends on the conda version and the installed Nvidia drivers. Most of the time it'll be CUDA 11 but it's not guaranteed. Anyway, yes, I'll move this to 11.2.